### ISSUE TITLE
ExecuteController read problem

### ISSUE THREAD

**Issue by AaronJing** [ucb-bar/gemmini#147]
Hi!

I found execute controller could read from normal scratchpads and accumulators. However, I did not find a test case that the ex_controller will actually read inputs from accumulators. When I write a test case myself, I load matrix A into accumulators and do the multiplication, however, it stalled. Could you suggest a correct test case, please?

Thanks!

**Comment by hngenc** [comment]
You're right; we don't currently have a test for that. In fact, we added a Config parameter to control whether or not the `ExecuteController` is even able to read data from the Accumulaor (the default setting is that it _can_ read from the accumulator):

https://github.com/ucb-bar/gemmini/blob/dev/src/main/scala/gemmini/Configs.scala#L142

Could you share your failing test here?
